### PR TITLE
Feature/clean tech debt upload fallback cx 2824

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Task
Clean tech debt added in `hotfix/release`
- Removed redundant error handling
- Removed function to return default for uploadFallback

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-525-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-525-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://525-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Router/index.js [hunk 1]
```diff
@@ -167,9 +167,18 @@ class MainRouter extends Component {
...
     const {documentType, options} = this.props
-    const {steps, token, language, uploadFallback} = options
+    const {steps, token, language} = options
+    const uploadFallback = this.uploadFallback(steps)
     const woopraCookie = getWoopraCookie()
-    return {steps, token, language, documentType, step: this.state.crossDeviceInitialStep, uploadFallback, woopraCookie}
+    return {steps, token, language, documentType, uploadFallback, step: this.state.crossDeviceInitialStep, woopraCookie}
+  }
+
+  uploadFallback = (steps) => {
```
Comment (rfreitas) [inline src/components/Router/index.js]: why are we still putting this on the root?

File: src/components/Capture/Face.js [hunk 2]
```diff
@@ -79,7 +86,7 @@ class Face extends Component {
           onCapture={ this.handleCapture }
         />
       :
-      this.props.uploadFallback ?
+      props.uploadFallback ?
```
Comment (stephencookdev) [inline src/components/Capture/Face.js]: Not strictly related to this PR - but this is a bit of a weird pattern, here. This `props` was created just to introduce a default prop of `onError: this.handleError`

Default props have a clear pattern already in React - doing this weird `props` thing instead is just confusing

Comment (stefaniacardenas) [reply src/components/Capture/Face.js]: You are right, it's ugly. I will change that as part of another ticket and make sure the pattern is not used elsewhere

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-525-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-525-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://525-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-525-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://525-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-525-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**